python/seldon_core/env_utils.py

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
Could we just have these in the utils.py? Is there a particular reason to have a separate utils file/

Reply:
There seems to be some circular dependencies if we do that
```
seldon_core/metrics.py:16: in <module>
    from seldon_core.utils import get_image_name, get_model_name, get_deployment_name, get_predictor_name, \
seldon_core/utils.py:16: in <module>
    from seldon_core.user_model import (
seldon_core/user_model.py:10: in <module>
    from seldon_core.metrics import SeldonMetrics, validate_metrics
```
 I have not investigated a lot into that though.  I think that having a separate module to have all env variables handling is ok. isnt it?

Reply:
Yeah, it's already an improvement to have them in one place so if there is no obvious solution to circular dependency let's leave it as it is :+1: